functional-tests/README.md

```diff
@@ -23,38 +23,62 @@ verify Che itself.
+```
+
+### Run tests via maven:
+By default, tests will execute against production environment (che.openshift.io).
+```
+mvn clean verify -Pfunctional-tests \
+    -Dche.testuser.name=<OSIO_USERNAME> \
+    -Dche.testuser.email=<OSIO_EMAIL> \
+    -Dche.testuser.offline_token=<OSIO_OFFLINE_TOKEN> \
+    -Dche.testuser.password=<OSIO_PASSWORD>
+```
+If you need to run tests against another environment (for example prod-preview), few more variables have to be set (and che-starter has to be started with different parameters). For example, running tests against `prod-preview` will look like this:
+```
+docker run -p 10000:10000 \
+    -e "GITHUB_TOKEN_URL=https://auth.prod-preview.openshift.io/api/token?for=https://github.com" \
+    -e "OPENSHIFT_TOKEN_URL=https://sso.prod-preview.openshift.io/auth/realms/fabric8/broker/openshift-v3/token" \
+    -e "CHE_SERVER_URL=https://che.prod-preview.openshift.io" \
+    registry.devshift.net/almighty/che-starter:latest
+
+mvn clean verify -Pfunctional-tests \
+    -Dche.testuser.name=<OSIO_PROD_PREVIEW_USERNAME> \
+    -Dche.testuser.email=<OSIO_PROD_PREVIEW__EMAIL> \
+    -Dche.testuser.offline_token=<OSIO_PROD_PREVIEW__OFFLINE_TOKEN> \
+    -Dche.testuser.password=<OSIO_PROD_PREVIEW__PASSWORD> \
+    -Dche.host=che.prod-preview.openshift.io \
+    -Dche.offline.to.access.token.exchange.endpoint=https://auth.prod-preview.openshift.io/api/token/refresh
+```
+
+### Full list of variables
+
+These tests require TestNG profile and listener must be set to `com.redhat.che.selenium.core.RhCheSeleniumTestHandler`.
+
+VM options (Mandatory parameters in __bold__)
+
+| Variable | Description |
 |----------|---------|
-| CHE_INFRASTRUCTURE | infrastructure on which Che runs (e.g. openshift) |
-| CHE_TESTUSER_EMAIL | email of testing user |
-| CHE_TESTUSER_OFFLINE__TOKEN | refresh token of testing user |
-| CHE_MULTIUSER | use multiuser mode |
-| CHE_OFFLINE_TO_ACCESS_TOKEN_EXCHANGE_ENDPOINT | endpoint for autentization (e.g. https://auth.openshift.io/api/token/refresh) |
-| CHE_TESTUSER_NAME | username of testing user |
-| CHE_TESTUSER_PASSWORD | password of testing user |
+| __che.testuser.email__ | Openshift.io e-mail |
+| __che.testuser.name__ | Openshift.io username |
+| __che.testuser.offline_token__ | Openshift.io offline token |
+| __che.testuser.password__ | Openshift.io password |
+| che.threads | Thread count. Default: `1` |
+| che.workspace_pool_size | Amount of workspace to be used while testing. Default: `1` |
+| che.host | Host where Che runs. Default: `che.openshift.io` |
+| che.port | Port where Che runs. Default: `443` |
+| che.protocol | Protocol. Default: `https` |
+| grid.mode | Used, when running with selenium-grid. Default: `false` |
+| browser | browser for selenium tests. Default: `GOOGLE_CHROME`) |
```

Comment:
There is an extra ")" . What about adding third column with "Default" values? Would it be better to read?

Reply:
Comments addressed.